### PR TITLE
close register for bsc relayers

### PR DESCRIPTION
**Pull request by yutianwu**
### Description
This PR will close the register interface on `relayerhub` temporarily.

### Rationale
The hacker register as a `relayer` and exploit the mekle proof vulnerability,  before we get further enhancement, we will close the
register interface temporarily.

### Example
NA

### Changes
NA

### CODE REVIEW

**Review by cosinlink — APPROVED** [review]

File: contracts/RelayerHub.sol [hunk 1]
```diff
@@ -56,13 +56,6 @@ contract RelayerHub is IRelayerHub, System, IParamSubscriber{
     alreadyInit = true;
   }
 
-  function register() external payable noExist onlyInit notContract noProxy{
```
Comment (Pythonberg1997) [inline contracts/RelayerHub.sol:59]: You have to change this part in template too

Comment (guagualvcha) [reply contracts/RelayerHub.sol:59]: Let us add a templated.

**Review by guagualvcha — DISMISSED** [review]

**Review by guagualvcha — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
-    "generate-test": "node generate-system.js --mock true && node generate-systemReward.js --mock true && node generate-validatorset.js --mock true && node generate-system.js --mock true && node generate-crosschain.js --mock true && node generate-tokenhub.js --mock true && node generate-tendermintlightclient.js --mock true && node generate-relayerincentivizecontract.js --roundSize 30 --maximumWeight 3 --mock true && node generate-slash.js --mock true",
+    "generate-test": "node generate-system.js --mock true && node generate-relayerhub.js --mock true && node generate-systemReward.js --mock true && node generate-validatorset.js --mock true && node generate-system.js --mock true && node generate-relayerhub.js --mock true && node generate-crosschain.js --mock true && node generate-tokenhub.js --mock true && node generate-tendermintlightclient.js --mock true && node generate-relayerincentivizecontract.js --roundSize 30 --maximumWeight 3 --mock true && node generate-slash.js --mock true",
```
Comment (cosinlink) [inline package.json:20]: this is for `testnet` while chainId is "714" from `genesis.json`, is it expected or changed on "genrate-QA" ?

**Review by cosinlink — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (owen-reorg) [inline package.json:20]: `generate-relayerhub.js` appears twice.
Maybe it's better to put this long command into a `.sh` script to make it clear.

Comment (guagualvcha) [reply package.json:20]: No, it is for UT.

Comment (guagualvcha) [reply package.json]: Fixed

**Review by guagualvcha — APPROVED** [review]

File: genesis.json [hunk 3]
```diff
@@ -31,46 +34,56 @@
...
...908181526020018383808284376000920191909152505060408051808201909152601d81527f63616c6c6572436f6d70656e736174696f6e44656e6f6d696e61746f7200000060208201529150611e4b9050565b1561136957602081146112d55760405162461bcd60e51b815260040180806020018281038252602e81526020018061224b602e913960400191505060405180910390fd5b604080516020601f840181900481028201810190925282815260009161131391858580838501838280828437600092019190915250611f3292505050565b9050801580159061132657506003548110155b6113615760405162461bcd60e51b81526004018080602001828103825260618152602001806121066061913960800191505060405180910390fd5b6004556114a5565b6113dd84848080601f0160208091040260200160405190810160405280939291908181526020018383808284376000920191909152505060408051808201909152601b81527f64796e616d69634578747261496e63656e74697665416d6f756e74000000000060208201529150611e4b9050565b1561146857602081146114215760405162461bcd60e51b815260040180806020018281038252602e815260200180612167602e913960400191505060405180910390fd5b604080516020601f840181900481028201810190925282815260009161145f91858580838501838280828437600092019190915250611f3292505050565b600e55506114a5565b6040805162461bcd60e51b815260206004820152600d60248201526c756e6b6e6f776e20706172616d60981b60
```
Comment (owen-reorg) [inline genesis.json:85]: no `0x` prefix for this account, does the code handle the compatibility?

Comment (guagualvcha) [reply genesis.json:85]: Yes.

**Review by owen-reorg — APPROVED** [review]

**Review by cosinlink — APPROVED** [review]

**Pull request merged**